unity/Assets/Scripts/AgentManager.cs

```diff
@@ -104,11 +104,11 @@ void Start()
         physicsSceneManager = GameObject.Find("PhysicsSceneManager").GetComponent<PhysicsSceneManager>();
 	}
 
-	private void initializePrimaryAgent() 
+	private void initializePrimaryAgent()
     {
```

Comment:
i did not clean up all the formatting issues... is there a reason the { are misaligned?
should we add linting?

Reply:
yes, we should. We are working on big backlog of features for different projects, let's talk about it next time we meet with Eric.